src/org/plovr/InputFileHandler.java

```diff
@@ -261,15 +271,29 @@ public String apply(JsInput input) {
   @Override
   protected void setCacheHeaders(Headers headers) {}
 
+  private void setAggressiveCacheHeaders(Headers headers) {
+    setDateHeader(headers);
+
+    Calendar cal = Calendar.getInstance();
+    cal.add(Calendar.SECOND, 315360000);
```

Comment:
for what it's worth, i think it's typical to set this to `31536000` (one year). Setting it to 10 years doesn't buy you anything because the browsers ignore it and truncate to one year (per https://www.ietf.org/rfc/rfc2616.txt)